src/components/Arrow-Right-Icon/arrow-right.tsx

```diff
@@ -1,15 +1,12 @@
-import StyledArrowRight, { StyledSvg } from "./arrow-right.style"
+import  { StyledSvg } from "./arrow-right.style"
 
 const ArrowRightIcon = () => {
     return (
-        <StyledArrowRight>
-            <StyledSvg width="17" height="14" viewBox="0 0 17 14" fill="none" xmlns="http://www.w3.org/2000/svg">
-                <path d="M15.9998 6.99991L0.999756 6.99991" stroke="white" strokeWidth="1.5" strokeLinecap="round" strokeLinejoin="round" />
-                <path d="M9.94995 0.975506L16 6.99951L9.94995 13.0245" stroke="white" strokeWidth="1.5" strokeLinecap="round" strokeLinejoin="round" />
-            </StyledSvg>
+        <StyledSvg width="17" height="14" viewBox="0 0 17 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
extract svg to asset

Reply:
resolved already in another pr - feature-side bar